ts/navigation/AppStackNavigator.tsx

```diff
@@ -50,10 +52,11 @@ type OnStateChangeStateType = Parameters<
   NonNullable<NavigationContainerProps["onStateChange"]>
 >[0];
 const isMainNavigatorReady = (state: OnStateChangeStateType) =>
-  state &&
-  state.routes &&
-  state.routes.length > 0 &&
-  state.routes[0].name === ROUTES.MAIN;
+  (state &&
+    state.routes &&
+    state.routes.length > 0 &&
+    state.routes[0].name === ROUTES.MAIN) ||
+  (state && state.routes[0].name === ITW_ROUTES.OFFLINE.WALLET);
```

Comment:
This is not always true, I think that checking for the main ITW navigator could be a more robust approach. Makes sense to you?

```suggestion
  (state && state.routes[0].name === ITW_ROUTES.MAIN);
```

Reply:
Done!